README.md

```diff
@@ -62,6 +62,29 @@ In order to start using `colourista` in your project, you will need to set it up
    main :: IO ()
    main = successMessage "All set up!"
    ```
+### Usage with Stack
+
+`colourista` is not available on Stackage yet, but fear not! You can still use
+it by adding the following to the `extra-deps` section of your `stack.yaml`
+file:
+
+```yaml
+extra-deps:
+  - github: kowainik/colourista
+    commit: c5fa55155a5915b1055d90ec632eebefb6b239fd
```

Comment:
As `colourista` is available on Hackage no need to use `github` sources for that.
It could be written as just 

```yaml
  - colourista-0.0.0.0
```

Reply:
What you are seeing here is my poor understanding of Stack. This works much better, thanks!